core/src/main/res/layout/noplayer_view.xml

```diff
@@ -1,5 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<com.google.android.exoplayer2.ui.SimpleExoPlayerView xmlns:android="http://schemas.android.com/apk/res/android"
-  android:id="@+id/simple_player_view"
+<com.google.android.exoplayer2.ui.AspectRatioFrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:id="@+id/video_frame"
   android:layout_width="match_parent"
-  android:layout_height="wrap_content" />
+  android:layout_height="wrap_content">
+
+  <SurfaceView
+    android:id="@+id/surface_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="center" />
+
+  <com.google.android.exoplayer2.ui.SubtitleView
```

Comment:
@zegnus I'll add it to my list, I plan to add a project view and start creating issues for things that we need in `no-player`